src/source/examples/post-campaign-v5-example.rst

```diff
@@ -0,0 +1,37 @@
+.. code-block:: javascript
+
+    POST /campaign
```

Comment:
we want to already announce it? Can we guarantee budgets always there when a campaign is created?

Reply:
apart from issues on our side regarding writing to multiple sources, yes